tests/RequestTest.php

```diff
@@ -103,6 +103,23 @@ public function testGetMetaDataWithPost()
         $this->assertSame(['request' => $data], $this->resolver->resolve()->getMetaData());
     }
 
+    public function testGetMetaDataWithGet()
+    {
+        $_SERVER['REQUEST_METHOD'] = 'GET';
```

Comment:
What would happen with lesser used request methods like `HEAD` or `PATCH`? Is it worth covering them?

Reply:
They don't have server variables and the data should be dropped into either the `json_decode` section due to `application/json` being present, or the `parse_str` section if not.  I'll do some manual testing on this to verify these cases are covered however.